go.mod

```diff
@@ -1,13 +1,12 @@
-module github.com/mailgun/mailgun-go/v3
+module github.com/kevineaton/mailgun-go/v3
```

Comment:
This should be `mailgun/` not `kevineaton`. I think `go get` might have rewritten this while you were working on this PR.